src/patternfly/components/Alert/alert.scss

```diff
@@ -39,6 +42,7 @@
   // Warning
   --pf-c-alert--m-warning__icon--Color: var(--pf-global--warning-color--200);
   --pf-c-alert--m-warning__icon--BackgroundColor: var(--pf-global--warning-color--100);
+  --pf-c-alert--m-warning__icon--FontSize: var(--pf-global--FontSize--xl);
```

Comment:
😢

Reply:
can you live with this? I can :)

Reply:
haha yeah. I just wish it didn't happen.